tooling/prettier/prettier-packs.js

```diff
@@ -0,0 +1,58 @@
+import fs from "fs";
```

Comment:
I spent a few hours trying to hack into the guts of `prettier`, but it was looking like I would have to restructure an unfamiliar AST. So, I gave up and made this simple post-processor script instead!